ingester/ingester.py

```diff
@@ -88,13 +92,126 @@ def standardize_trees_name(input_data, trees_name):
     return input_data
 
 
-def ingest_submissions(spool_dir, trees_name, db_client=None):
+def upload_logspec(logspec, id):
```

Comment:
Instead of using id in the name of the file, I'd generate a hash of the excerpt and use that. This way, we can have the same file for tests that have exactly the same excerpt.

Reply:
Excellent idea! Will do